pyconcz/templates/default/proposals/proposal_success.html

```diff
@@ -22,9 +22,13 @@ <h1>Application sent</h1>
             <h1>Proposal sent</h1>
 
             <p class="lead text-center">Thank you! <br>We’ve received your proposal.</p>
+
+            <p class="text-center font-weight-bold">Please note that you won’t receive any copy or confirmation of your proposal to your mailbox.</p>
+
             <p class="text-center">
                 All submissions will be scored after Call for Papers ends.
-                <br>The whole process takes some time and after it ends we let you know to the e-mail you’ve provided.
+                <br>The whole selection process takes usualy days.
```

Comment:
and starts after the deadline?

Reply:
Good point. It should be clear that the whole process may start after the deadline so nobody will expect some reply sooner.